pages/api/search/[type].ts

```diff
@@ -1,13 +1,15 @@
 import { NextApiRequest, NextApiResponse } from 'next';
 import  prisma  from '../../../lib/prisma';
-
+import { Behavior, User, Session, Program } from '@prisma/client'; 
```

Comment:
NIT: organize imports

Can do this in Visual Studio by right clicking on the code --> Source Action --> Organize imports

Should probably do this for all changed files actually.

Reply:
Good to know!